src/styles/blocks/buy-link.scss

```diff
@@ -0,0 +1,20 @@
+.buy-link {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 170px;
```

Comment:
```suggestion

```
Redundant because your button is a flex item and the card has paddings 14px. All available width is occupied